src/distdl/nn/mixins/conv_mixin.py

```diff
@@ -35,15 +32,7 @@ def _compute_min_input_range(self,
 
         """
 
-        # incorrect, does not take stride and dilation into account
-        # padding might also not be correct in these cases...
-        kernel_offsets = (kernel_size - 1) / 2
-
-        # for even sized kernels, always shortchange the left side
-        kernel_offsets[kernel_size % 2 == 0] -= 1
-
-        bases = idx + kernel_offsets - padding
-        return bases - kernel_offsets
+        return stride * idx - padding
```

Comment:
Does this break the existing layers?

Reply:
No, I will include a brief proof outline in the PR description